src/components/pill/__snapshots__/Pill.test.js.snap

```diff
@@ -0,0 +1,24 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
@TimeBones Did you re-run the tests? I see lots of tests but the snapshot is pretty minimal...